package.json

```diff
@@ -4,20 +4,21 @@
   "license": "MIT",
   "scripts": {
     "build": "npm run clean && mkdirp build && webpack --progress --colors",
+    "build:esmodules": "tsc --project ./tsconfig.json --module esnext",
     "build:demos": "npm run clean && mkdirp build && webpack --config=demos/webpack.config.js --progress --colors",
     "build:min": "mkdirp build && cross-env MDC_ENV=production webpack -p --progress --colors",
     "changelog": "standard-changelog -i CHANGELOG.md -k packages/material-components-web/package.json",
-    "clean": "del-cli build/** build .rewrite-tmp/** .rewrite-tmp",
-    "clean:site": "del-cli .site-generator-tmp/** .site-generator-tmp",
-    "dist": "npm run build && npm run build:min",
+    "clean": "node ./scripts/build/clean",
+    "clean:site": "node -e \"require('./scripts/build/clean').removeDirectory('.site-generator-tmp')\"",
```

Comment:
Can we pass a flag to `scripts/build/clean` instead of inlining JS code here?

E.g.:

```
"clean:site": "node ./scripts/build/clean --site-generator-only",
```